util/rbac/rbac.go

```diff
@@ -468,7 +468,11 @@ func loadPolicyLine(line string, model model.Model) error {
 		return err
 	}
 
-	if len(tokens) < 2 || len(tokens[0]) < 1 {
+	tokenLen := len(tokens)
```

Comment:
I think we need to re-introduce `len(tokens[0]) < 1`, otherwise we're no longer checking for a zero-length first field.

Reply:
It is here: https://github.com/argoproj/argo-cd/pull/11964/files#diff-5aa2104a607d988c61381c689d286efd5041c938b46709ae2655a3286beb88bdR473

Invalid format triggers on those conditions:
```go
tokenLen < 1 ||
		(tokens[0] == "g" && tokenLen != 3) ||
		(tokens[0] == "p" && tokenLen != 6) 
```

Reply:
@crenshaw-dev bump

Reply:
@zekth apologies for the slow review!

I think we still have a problem if `tokens[0] == ""`. As I read it, that input satisfies all conditions and is treated as valid.

Reply:
Oh i misread your statement, i read `tokenLen < 1` and not `tokens[0]`. Fixing it